src/test/java/laoflch/debezium/connector/informix/InformixFixColumnOrderIT.java

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright Debezium-Informix-Connector Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package laoflch.debezium.connector.informix;
+
+import static laoflch.debezium.connector.informix.InformixConnectorConfig.SNAPSHOT_MODE;
+import static laoflch.debezium.connector.informix.InformixConnectorConfig.SnapshotMode.INITIAL_SCHEMA_ONLY;
+import static laoflch.debezium.connector.informix.util.TestHelper.TEST_DATABASE;
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.sql.SQLException;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.data.VerifyRecord;
+import io.debezium.embedded.AbstractConnectorTest;
+
+import laoflch.debezium.connector.informix.util.TestHelper;
+
+public class InformixFixColumnOrderIT extends AbstractConnectorTest {
+
+    private InformixConnection connection;
+    private static final String testTableName = "test_table";
```

Comment:
Please use a specific table name for each test case, instead of `test_table`. So that in `testdb` database, we can easily find which table is for which test case.